src/uct/api/uct_def.h

```diff
@@ -145,7 +147,7 @@ typedef struct uct_iov {
  * @param [in]  data     Points to the received data. This may be a part of
  *                       a descriptor which may be released later.
  * @param [in]  length   Length of data.
- * @param [in]  flags    Mask with @ref uct_cb_flags
+ * @param [in]  flags    Mask with @ref uct_am_cb_flags
```

Comment:
Along with this change, shouldn't UCT_CB_FLAG_DESC be renamed to UCT_AM_CB_FLAG_DESC?

Reply:
yes